components/PageControl/examples/PageControlTypicalUseExample.m

```diff
@@ -149,4 +162,13 @@ + (NSDictionary *)catalogMetadata {
   };
 }
 
+- (CGFloat)offsetMultiplierForPage:(NSInteger)page numberOfPages:(NSInteger)numberOfPages {
```

Comment:
I still needed an `offsetMultiplier` [here](https://github.com/material-components/material-components-ios/pull/5711/files#diff-38e7b891cbe473f025c921800618dd7fR61) but otherwise took your suggestions in fa63c28